applications/nrf_desktop/src/hw_interface/selector_hw.c

```diff
@@ -269,16 +273,10 @@ static int init(void)
 	BUILD_ASSERT(ARRAY_SIZE(selector_config) > 0,
 			 "There is no active selector");
 
-	for (size_t i = 0; i < ARRAY_SIZE(port_map); i++) {
-		if (!port_map[i]) {
-			continue;
-		}
-
-		gpio_dev[i] = device_get_binding(port_map[i]);
-
-		if (!gpio_dev[i]) {
-			LOG_ERR("Cannot get GPIO%u dev binding", i);
-			return -ENXIO;
+	for (size_t i = 0; i < ARRAY_SIZE(gpio_dev); i++) {
+		if (!device_is_ready(gpio_dev[i])) {
```

Comment:
I think we could allow to use selector on devices that e.g. do not use GPIO1 (e.g. if we only connect to pins of GPIO0). Currently we use the module only for nrf52840 based gaming mouse reference design, but customers are free to port it to e.g. smaller SoC if needed.